src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int LOTTERY_MAX_COUNT = 100;
+    private final Random random = new Random();

```

Comment:
The `LOTTERY_MAX_COUNT` constant should be declared static since it stays constant throughout all instances of the class. Use `private static final int LOTTERY_MAX_COUNT = 100;` instead. 🛠️